docs/features/bank.md

```diff
@@ -0,0 +1,197 @@
+                {
+                  "address": "faa1ydhmma8l4m9dygsh7l08fgrwka6yczs0gkfnvd",
+                  "coins": [
+                    {
+                      "denom": "iris-atto",
+                      "amount": "100000000000000000000"
+                    }
+                  ]
+                }
+              ],
+              "outputs": [
+                {
+                  "address": "faa1ut8aues05kq0nkcj3lzkyhk7eyfasrdfnf7wph",
+                  "coins": [
+                    {
+                      "denom": "iris-atto",
+                      "amount": "100000000000000000000"
+                    }
+                  ]
+                }
+              ]
+            }
+          }
+        ],
+        "fee": {
+          "amount": [
+            {
+              "denom": "iris-atto",
+              "amount": "40000000000000000"
+            }
+          ],
+          "gas": "200000"
+        },
+        "signatures": null,
+        "memo": ""
+      }
+    }
+    ```
+    Save the result to a file.
+    
+    Send signature transaction:
+    ```bash
+    iriscli bank sign [file] --chain-id=[chain-id]  --name [key name of from account] 
+    ```
+    Return signed transactions:
+    ```json
+    {
+      "type": "auth/StdTx",
+      "value": {
+        "msg": [
+          {
+            "type": "cosmos-sdk/Send",
+            "value": {
+              "inputs": [
+                {
+                  "address": "faa1ydhmma8l4m9dygsh7l08fgrwka6yczs0gkfnvd",
+                  "coins": [
+                    {
+                      "denom": "iris-atto",
+                      "amount": "100000000000000000000"
+                    }
+                  ]
+                }
+              ],
+              "outputs": [
+                {
+                  "address": "faa1ut8aues05kq0nkcj3lzkyhk7eyfasrdfnf7wph",
+                  "coins": [
+                    {
+                      "denom": "iris-atto",
+                      "amount": "100000000000000000000"
+                    }
+                  ]
+                }
+              ]
+            }
+          }
+        ],
+        "fee": {
+          "amount": [
+            {
+              "denom": "iris-atto",
+              "amount": "40000000000000000"
+            }
+          ],
+          "gas": "200000"
+        },
+        "signatures": [
+          {
+            "pub_key": {
+              "type": "tendermint/PubKeySecp256k1",
+              "value": "A+qXW5isQDb7blT/KwEgQHepji8RfpzIstkHpKoZq0kr"
+            },
+            "signature": "5hxk/R81SWmKAGi4kTW2OapezQZpp6zEnaJbVcyDiWRfgBm4Uejq8+CDk6uzk0aFSgAZzz06E014UkgGpelU7w==",
+            "account_number": "0",
+            "sequence": "11"
+          }
+        ],
+        "memo": ""
+      }
+    }
+    ```
+    Save the result to a file.
+    
+5. Broadcast transactions
+
+    Broadcast signed transactions generated offline, using the final generated file in step 4:
+    ```bash
```

Comment:
> Broadcast signed transactions generated offline, using the final generated file in step 4:
How about change the sentence like this:
```
Broadcast offline signed transaction. Here you just use the transaction generated by above sign command. Of course, you can generate your signed transaction by any methods. Just make sure your signature and public key are amino encoding.
```

Reply:
Great！